### PR TITLE
chore(blog): Add V1 migration blog post

### PR DESCRIPTION
**Pull request by hermanschaaf**
A blog post to summarize the major user-facing V1 changes and help users migrate from v0.

### CODE REVIEW

**Comment by yevgenypats** [comment]
That's amazing! I'll review shortly.

File: website/pages/blog/migrating-from-v0-to-v1.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+The [announcement blog post](/blog/cloudquery-v1-release) lists many of the important improvements, and we won't re-iterate them all here. Most changes are internal and developer-facing, but some do impact existing CloudQuery teams. Those are:
+- [changes to CLI commands](#changes-to-the-cli-commands)
+- [changes to the configuration format](#changes-to-the-configuration-format)
+- [changes to tables and schemas](#changes-to-tables-and-schemas)
+
+## Changes to the CLI Commands
+
+Users of CloudQuery v0 would be familiar with the main commands `init` and `fetch`, as well as some others. These are no longer in V1, but similar commands have replaced them to reflect that CloudQuery now supports both source and destination plugins.
+
+### Generate commands
```
Comment (yevgenypats) [inline website/pages/blog/migrating-from-v0-to-v1.md:27]: Let's add this section once we add `gen` command back

**Review by yevgenypats — APPROVED** [review]

File: website/pages/blog/migrating-from-v0-to-v1.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+### Sync commands
+
+`cloudquery sync` replaces the v0 `cloudquery fetch` command.
+
+Functionally it is still the same: it loads data from a source to a destination, but `sync` now supports multiple destinations, while `fetch` only supported PostgreSQL. With this change also comes a change in expected config format, see the [next section](#changes-to-the-configuration-format) for more details on this.
+
+`cloudquery sync` needs to be passed a path to a config file or directory containing config files. So for example, to sync using a single yaml file:
+
+```
+cloudquery sync config.yml
```
Comment (erezrokah) [inline website/pages/blog/migrating-from-v0-to-v1.md:43]: The guide looks great, but I'm curious how this works. Don't we need at least 2 configuration files (1 for a source and 1 for a destination)?

Comment (hermanschaaf) [reply website/pages/blog/migrating-from-v0-to-v1.md:43]: Yeah, it's explained a little bit lower down in the post, but probably should be mentioned here as well: you will need to separate the different parts of the config with `---`

**Pull request merged**